Lib/test/test_py_compile.py

```diff
@@ -114,7 +114,8 @@ def test_relative_path(self):
         self.assertFalse(os.path.exists(self.cache_path))
 
     # TODO: RUSTPYTHON
-    if sys.platform == "darwin":
+    import platform
+    if sys.platform == "darwin" and platform.release() != "20.3.0":
```

Comment:
```python
tuple(int(p) for p in platform.release().split('.')) >= "20.3.0"
```

Reply:
Wouldn't it be `>= (20, 3, 0)` ?

Reply:
@youknowone Does 84f13e1 work on your architecture?

Reply:
Yes, it does work :+1: